recipes/biotite/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "0.12.0" %}
+{% set number = "0" %}
+{% set sha256 = "06ed9ec5dc14d3ee112ed9862c7738ad83c6e24022bc9a4e8646580fb7196207" %}
+
+package:
+  name: biotite
+  version: {{ version }}
+
+source:
+  url: https://github.com/biotite-dev/biotite/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ number }}
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  skip: True  # [py < 36]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python 3.6
```

You shouldn't constrain the python version here as you've stated above that it should be >= 3.6. There will then be two builds for each platform one for Python 3.6 and one for Python 3.7.
```suggestion
    - python
```
This kind of version specifier for `python` should only be used for `noarch: python` packages.